VirtoCommerce.CustomerModule.Web/Controllers/Api/CustomerModuleController.cs

```diff
@@ -351,14 +351,14 @@ public IHttpActionResult GetOrganizationById(string id, [FromUri]string response
         }
 
         /// <summary>
-        /// Get plenty organizations 
+        /// Get plenty organizations
         /// </summary>
         /// <param name="ids">Organization ids</param>
         /// <param name="responseGroup">Response group flags controls fullness of resulting object graph</param>
-        [HttpGet]
-        [Route("organizations")]
+        [HttpPost]
```

Comment:
Agreed on the previous comment, we could not remove the existing method from API to save backward compatibility.
You can add POST, not to replace the existing one.